resource/resource_test.go

```diff
@@ -95,7 +95,8 @@ func TestIn(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			r := Resource{}
 
-			version, metadata, err := r.In("/tmp", defSource, oc.Params{}, tc.inVersion, defEnv, silentLog)
+			version, metadata, err := r.In("/tmp", defSource, oc.Params{},
+				tc.inVersion, defEnv, silentLog)
```

Comment:
It looks like I'm now really appreciating the way Black formats python code. The following would be much better IMO:
```suggestion
			version, metadata, err := r.In(
				"/tmp", defSource, oc.Params{}, tc.inVersion, defEnv, silentLog
			)
```

Reply:
The problem is that this is not enforceable by any tool, so it would drift.
The root cause is that gofmt, on purpose, leaves the decision about line length to the author, and 3rd-party tools are not good enough :-(
Also, I don't particularly like this specific Black decision, the power of gofmt is that it comes with the language and there should be no discussions about its output, but the line length kicks in :-(

Reply:
I understand. Doing the split between the function name and its argument is one of the decision that Black took that I really appreciate. It's one of the thing that really make the code more readable. I'm less happy about it's willingness to split the arguments one per line if they don't all fit on the same line.

Reply:
At the end I followed this Back thing.